README.md

```diff
@@ -24,13 +24,15 @@ Download and install node. Go to https://nodejs.org/en/download/ and follow the
 
 Once you have Docker and Node installed and setup then do the following:
 
-    copy server/src/configs/sample.config.ts -> server/src/configs/config.ts
+    cp server/sample-bluebutton-config.json -> server/.bluebutton-config.json
+
+or (if running a docker compose selenium tests)
+
+    cp server/sample-bluebutton-selenium-config.json -> server/.bluebutton-config.json
```

Comment:
@JFU-GIT Should we mention selenium tests in the main README?  These are intended for internal team usage. These should be moved to the README-bb2-dev.md.